pex/pex_bootstrapper.py

```diff
@@ -559,7 +559,8 @@ def bootstrap_pex(entry_point):
                 venv_pex = ensure_venv(pex.PEX(entry_point, interpreter=target))
             except ValueError as e:
                 die(str(e))
-            os.execv(venv_pex, [venv_pex] + sys.argv[1:])
+            venv_python = os.path.join(os.path.dirname(venv_pex), "bin", "python")
```

Comment:
Nice! Is the non-version specific symlink guaranteed by virtualenv to exist?

Reply:
Yup.